src/inputs/internal/ComboBoxBase.tsx

```diff
@@ -419,7 +428,8 @@ type FieldState<O> = {
   allOptions: O[];
   optionsLoading: boolean;
 };
-export type OptionsOrLoad<O> = O[] | { initial: O[]; load: () => Promise<{ options: O[] }> };
+type LoadOption<O> = { initial: O[]; load: () => Promise<{ options: O[] }> };
+export type OptionsOrLoad<O> = O[] | LoadOption<O>;
```

Comment:
**question/nit:** Any reason to extract out this type?

Reply:
Actually no, I did it in order to make some tests, I just revert it on last commit.